src/Pages/Authentication/Login.jsx

```diff
@@ -0,0 +1,78 @@
+import "../../css/main.css";
+import "./authentication.css";
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { useAuth } from "../../Context";
+
+export const Login = () => {
+
+    const [userDetails, setUserDetails] = useState({email: "", password: "", rememberMe: false})
+    const [error, setError] = useState({isError: false, text: ""})
+    const [showPasswordToggle, setShowPasswordToggle] = useState(true)
+    const { login, testLogin } = useAuth();
+    useEffect(() => {
+        const timeoutId = setTimeout(() => {
+            setError({...error, isError: false})
+        }, 3000);
+
+        return () => clearTimeout(timeoutId);
+    },[error])
+
+    const submitHandler = () => {
+
+        const passwordValidation = /^(?=.*\d)(?=.*[a-z]).{5,10}$/;
```

Comment:
suggestion - I think password validation isn't required on login, if you're doing it on signup.

Reply:
yes correct thanks for suggesting this will remove the password validation here.